app/javascript/pages/orientation/Carousel.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <section>
+        <b-carousel
+            :autoplay="auto"
+            :arrow="arrow"
+            :repeat="arrowBoth"
+            :arrow-hover="arrowHover"
+            :interval="interval"
+            :pause-hover="pauseHover">
+            <b-carousel-item v-for="(announcement, i) in announcements" :key="i" :id="announcement.id">
+                <section :class="`hero is-light`" style="background-color:grey-lightest">
+                    <div class="hero-body has-text-centered">
+                        <h1 class="title"  v-if="announcement.id === 'default'">{{announcement.name}}</h1>
+                        <h1 class="title"  v-else>
+                            <a :href="`/announcements/${announcement.id}`">{{announcement.name}}</a>
+                        </h1>
+                        <p class="content">{{announcement.description}}</p>
+                    </div>
+                </section>
+            </b-carousel-item>
+        </b-carousel>
+    </section>
+</template>
+
+<script>
+import { Carousel } from 'buefy';
+const fetch = require("node-fetch");
+
+export default {
+    data() {
+        return {
+            auto: true,
+            arrow: true,
+            pauseHover: false,
+            arrowBoth: true,
+            arrowHover: true,
+            interval: 5000,
```

Comment:
Minor: `data` is typically used to setup up 'observed' variables, so that whenever one changes, vue knows to track anywhere it's used and propagate the change. IIUC, the values above are static and so don't technically need to be in `data`. A couple of alternatives are to hardcode them directly into the view (probably the simplest solution) or, if you want to make it more apparent what the available options are, define them as [`$options`](https://vuejs.org/v2/api/#vm-options).